_contributors-app/swap1108.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Swapnil Sourabh
+description: Programmer in the beta phase ! 
+picUrl: https://avatars1.githubusercontent.com/u/26069854?s=400&u=619847deb70dab00c2334e72aa18e0d2d33b0905&v=4
+skills: |
+  - **Languages:** Python, C/C++, Java
+  - **Tools:** git, Github, Latex, Putty
+
+githubUsername: swap1108
+speciality: |
+
+  - Machine Learning
+
+  - Internet of Things
+
+  - Data Structures and Algorithms
+
+  - Web Development
+
+
+github_url: https://github.com/swap1108
+linkedIn_url: https://www.linkedin.com/in/swapnil-sourabh-658a3a147/
+twitter_url: https://twitter.com/swapnilsourabh
+
+work_experience: |
+ - Research Intern @ National University of Singapore, Singapore [June 2018 - Present]
+
+ - Former Summer Intern @ Indian Institute of Technology, Patna [ May 2017- July 2107]
+
+ - owner @ algoskills
+
+project:
+- |
+   - Algokills- A website that provides an interactive platform for computer algorithms. A step taken to learn and contribute (https://www.algoskills.com)
+
+- |
+    - Modification of Ranking based mutation in Differential Evolution (present)
+
+- |
+    - Activity recognition using smartphone sensors with auto-annotation in IoT: An innovative way for human acitvity recognition.
+
+- |
+    - Smart Home: ’Prediction of resident’s activities for smart living!’ A Concept derived from assisted living.
+
+- |
+    - Personal Financial Assistant: ’A Java desktop app which acts as a personal financial assistant!’· It provides an interface for users to save their financial data securely and manage the assets.
+
+About me: |
```

Comment:
Do we have an **About me** field @knrt10 ?

Reply:
If you want to add anything extra other than in template, you can add after the end of
```
---
```
line. cc @swap1108